### ISSUE TITLE
Cursor color turns black

### ISSUE THREAD

**Issue by zbyju** [SmiteshP/nvim-navbuddy#88]
Once I open Navbuddy (`:Navbuddy`) my cursor turns black for the remainder of using my terminal (even outside neovim).

1. Cursor is normal:
![](https://i.imgur.com/bXtAJyx.png)
2. Open navbuddy
![](https://i.imgur.com/obDpKlu.png)
3. Cursor is black
![](https://i.imgur.com/Ys278gG.png)
4. Even outside neovim
![](https://i.imgur.com/l83Mw9G.png)

I'm using Windows 10 + WSL2 and it happens both in Windows Terminal and Wezterm. I changed my font which didn't have an effect.

My config is using packer:
```lua
use {
  "SmiteshP/nvim-navbuddy",
  requires = {
    "neovim/nvim-lspconfig",
    "SmiteshP/nvim-navic",
    "MunifTanjim/nui.nvim",
    "numToStr/Comment.nvim",        -- Optional
    "nvim-telescope/telescope.nvim" -- Optional
  }
}
```

Mason lsp:
```lua
on_attach = function(client, bufnr)
  navic.attach(client, bufnr)
end
```

Navbuddy setup:
```lua
local navbuddy = require("nvim-navbuddy")
local actions = require("nvim-navbuddy.actions")
navbuddy.setup {
  lsp = {
    auto_attach = true,
  },
}
```

*BTW: I'm quite new to neovim, but I've never seen such issue*

**Comment by zbyju** [comment]
Also: this doesn't happen with any other plugin - Telescope, Trouble, etc. all work fine; only when going out of Navbuddy does the cursor change.

**Comment by lljbash** [comment]
Same here, also with WSL2+WT

**Comment by SmiteshP** [comment]
Even outside neovim!? Is this specific to WSL only?

**Comment by SmiteshP** [comment]
Yeah I was also able to replicate this issue on WSL2, seems like setting guicursor with highlight group that has "blend" option set is messing things up somehow. But since this persists even outside neovim this might be a WSL releated bug 🤔 or maybe neovim bug, no idea though

**Comment by SmiteshP** [comment]
Maybe we file a bug in neovim repo itself ?

**Comment by ktkimit** [comment]
Same here, but only on WT + wsl2.
